### ISSUE TITLE
FR: Custom codeblock to display bibliography in file

### ISSUE THREAD

**Issue by lguenth** [mgmeyers/obsidian-pandoc-reference-list#42]
Hey! 
Yesterday, [some people on Discord](https://discord.com/channels/686053708261228577/722584061087842365/1000885280175300618) were discussing different options to display a dynamically updated bibliography as part of the opened document, similar to how the Word integration for Zotero works. 
I thought it might be a great addition to your plugin to include a similar funcitonality, perhaps as a custom codeblock, similar to how Dataview and Breadcrumbs display and update results.
This would likely only be usable for drafts and notes since it would break in an export with Pandoc but, at least for me, it would be really helpful if I could close all sidebars, go into focus mode and still see the bibliography.

**Comment by chrisgrieser** [comment]
for the code block format, I'd suggest two options:
1) normal code block:
````
```refs
```
````
2) directly use pandoc's placeholder for references
```
::: {#refs}
:::
```

**Comment by FeralFlora** [comment]
This would be a great feature!

**Comment by mo-seph** [comment]
Agreed, this would be fantastic! As a use case, I often want to send people a PDF of my notes - with the references in the sidebar, there's no way to include them in that.

**Comment by ThyrixYang** [comment]
Any progress on this feature? I really love this idea.
